lib/testLauncher/ipc/server.js

```diff
@@ -27,24 +28,27 @@ const handleMessageFromClient = handleMessage(state);
 const start = config => {
 	const server = net.createServer();
 
+	state.server = server;
+
 	server.on('connection', socket => {
 		socket.id = `con#${++idCount}`;
 		state.connections[socket.id] = socket;
-		socket.on('data', handleMessageFromClient);
+		socket.on('error', err => logger.debug('IPC server socket error', err.errno));
 		socket.on('close', () => delete state.connections[socket.id]);
+		socket.on('data', handleMessageFromClient);
 		socket.unref();
 		write(socket, messageId.SETUP_SESSION, {config});
 	});
 
 	server.on('error', err => {
 		if (err.errno === 'EADDRINUSE') {
```

Comment:
When we can have this message? Does it makes sense for user to know about it as there is no way user can specified port or know in advance what port will be used by launcher

Reply:
I really don't know how to reproduce this error. IPC server should be started on any available port.

Reply:
So maybe we can have a message that all ports are busy or node cannot find available port?

Reply:
I don't know how to reproduce any of these scenarios, so I can't say which message would be better. We can try to investigate.